.ci/install.sh

```diff
@@ -2,10 +2,19 @@
 
 set -e
 
-INSTALL_DIR="$HOME/.fnm"
 RELEASE="latest"
 OS="$(uname -s)"
 
+if [ -d "$HOME/.fnm" ]; then
+  INSTALL_DIR="$HOME/.fnm"
+elif [ ! -z $XDG_DATA_HOME ]; then
```

Comment:
You can use `test -n` and double quotes.
```bash
elif [ -n "$XDG_DATA_HOME" ]; then
```

Reply:
Thanks for the tips!

I did try `test -n` before, but I tried it without double quotes:

```
elif [ -n $XDG_DATA_HOME ]; then
```

and in some edge case (e.g. `XDG_DATA_HOME=""`), this will cause unexpected result.

Reply:
Now I understand that string wrapped by double quotes is very different that without double quotes in bash,
and it's important to know when to wrap string by double quotes or not.